Lib/imaplib.py

```diff
@@ -19,10 +19,22 @@
 # GET/SETQUOTA contributed by Andreas Zeidler <az@kreativkombinat.de> June 2002.
 # PROXYAUTH contributed by Rick Holbert <holbert.13@osu.edu> November 2002.
 # GET/SETANNOTATION contributed by Tomas Lindroos <skitta@abo.fi> June 2005.
-
-__version__ = "2.58"
-
-import binascii, errno, random, re, socket, subprocess, sys, time, calendar
+# IDLE contributed by Forest <forestix@nom.one> August 2024.
+
+__version__ = "2.59"
+
+import binascii
+import calendar
+import errno
+import functools
+import platform
+import random
+import re
+import selectors
+import socket
+import subprocess
+import sys
+import time
```

Comment:
Generally, it's a bad idea to unrelated refactor code in a PR, as it makes the diff look much bigger than it should. I think this is a worthy change here though (just leaving this as a note for another reviewer).